<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="stack_main">
     <widget class="QPushButton" name="btn_test">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>test</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_msg_time">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stack_login">
     <widget class="QLineEdit" name="edit_login_pw">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>비밀번호</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_main_to_regist">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>회원가입</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_login_id">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>아이디</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>70</width>
        <height>55</height>
       </rect>
      </property>
      <property name="text">
       <string>로그인</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_main_to_find">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>아이디/비밀번호 찾기</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stack_regist"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
